mlflow/sentence_transformers/__init__.py

```diff
@@ -310,7 +346,8 @@ def _load_pyfunc(path):
     import sentence_transformers
 
     model = sentence_transformers.SentenceTransformer.load(path)
-    return _SentenceTransformerModelWrapper(model)
+    task = model_config["task"] if model_config else None
```

Comment:
```suggestion
    model_config = model_config or {}
    task = model_config.get("task", None)
```

Key safety in case if other use case for model config is added.